dojo/tools/checkmarx/parser.py

```diff
@@ -171,7 +171,6 @@ def _get_findings_xml(self, filename, test):
             # consolidate vuln_ids_from_tool values
             if self.mode != "detailed":
                 for key in list(dupes):
-                    vuln_ids_from_tool[key].sort
```

Comment:
This one may need to stay here. Sorting a list in python does not return a result, but updates the list in memory